src/io.h

```diff
@@ -77,4 +78,22 @@ char *FGetsError(FILE * restrict file,const char * restrict fname,size_t *line,c
 size_t SkipNLines(FILE * restrict file,const size_t n);
 size_t SkipComments(FILE * restrict file);
 
+#ifdef POSIX
```

Comment:
ANSI escape codes supported in Unix-like since about 1984. 
POSIX developed as API for Unix-like systems and started since 1988 (also POSIX available on Windows, for example via 'cygwin').
So POSIX terminal (as abstraction) should support ANSI escape codes by design, but I cant find something about this in POSIX documentation http://www.open-std.org/jtc1/sc22/open/n4217.pdf

We can replace #ifdef POSIX -> #ifdef __unix__ and  don`t worry about it.

Reply:
![image](https://user-images.githubusercontent.com/13792919/46612991-46a70180-cb12-11e8-94c4-ea49e98e7d4d.png)